### ISSUE TITLE
Download redirects to NSFW

### ISSUE THREAD

**Issue by dada513** [Sildurs-shaders/sildurs-shaders.github.io#162]
Your great adfoc now redirects to NSFW sites. Yes redirects, not shows adverts, redirects. Also sometimes it redirects to scams. Stop using it

**Comment by CKenJa** [comment]
I also get annoying sites and apple calendar hacks when I click anywhere, I need to scrutinize it once to see if it has any weird commits.And hexa?

**Comment by dada513** [comment]
From my experience it seems to mostly be on adblock users but occasionally the redirect happens without it

**Comment by NaiNonTH** [comment]
Yeah, sometimes it just redirects into some kinda sussy downloading website.

**Comment by EinfxchFinn** [comment]
![Bildschirmfoto 2021-10-06 um 15 06 40](https://user-images.githubusercontent.com/58655519/136208386-a264619f-999c-4ef3-940c-3ddb2de329e4.png)
I also get redirected when I click on one of these links
